src/test/java/jenkins/plugins/logstash/persistence/BuildDataTest.java

```diff
@@ -130,11 +135,12 @@ public void after() throws Exception {
   private void verifyMocks() throws Exception
   {
     verify(mockProject).getName();
-    verify(mockProject).getFullName();
+    verify(mockProject, atLeastOnce()).getFullName();
+    verify(mockProject, atMostOnce()).getUrl();
 
     verify(mockBuild).getId();
     verify(mockBuild, times(2)).getResult();
-    verify(mockBuild, times(2)).getParent();
+    verify(mockBuild, atLeast(2)).getParent();
```

Comment:
I used `atLeast` instead of a fixed value here, because the amount of calls varies depending on the value returned by `mockBuild.getCauses`, therefore it depends on the exact contents of the test